### ISSUE TITLE
Remove hardcoded certificate details from chart

### ISSUE THREAD

**Issue by praveendhac** [controlplaneio/kubesec-webhook#6]
Remove hardcoded certificate details from helm chart, can be achieved using `genSignedCert`

**Comment by praveendhac** [comment]
helm command and status
```
helm install --namespace pd-test-my-chart2 --debug ./kubesec-webhook -n kubesec-pd2

$ helm ls
NAME       	REVISION	UPDATED                 	STATUS  	CHART                	APP VERSION	NAMESPACE
kubesec-pd2	1       	Sun Sep 22 02:06:52 2019	DEPLOYED	kubesec-webhook-0.1.0	1.0        	pd-test-my-chart2
```

Pod Status
```
$ kubectl get all -n pd-test-my-chart2
NAME                                               READY   STATUS    RESTARTS   AGE
pod/kubesec-pd2-kubesec-webhook-77d469d5b9-gvxp5   1/1     Running   0          6m39s

NAME                      TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)   AGE
service/kubesec-webhook   ClusterIP   10.106.193.244   <none>        443/TCP   6m39s

NAME                                          READY   UP-TO-DATE   AVAILABLE   AGE
deployment.apps/kubesec-pd2-kubesec-webhook   1/1     1            1           6m39s

NAME                                                     DESIRED   CURRENT   READY   AGE
replicaset.apps/kubesec-pd2-kubesec-webhook-77d469d5b9   1         1         1       6m39s
```
Verified helm chart, k8s secrets use `caKeyData/certData` values if configured in values.yaml  otherwise generate kyes/cert using `genCA` and `genSignedCert`.
Refer http://masterminds.github.io/sprig/crypto.html

**Comment by sublimino** [comment]
Closed in #7
